item_size_calculator/README.md

```diff
@@ -62,31 +67,31 @@ const item = {
             "S": "Book"
         }
     }
-
 ```
 
 ### **Calculate Size**
+
+```js
+const size = CALC.CalculateSize(item);
 ```
-const size =  CALC.CalculateSize(item);
-```
-```
+
+```js
 { 
     rcu: 1, 
     wcu: 1, 
-    size: 137 
+    size: 137 // in Bytes
 }
 ```
 
-### **Understand if an item is under the 400Kb limit**
-```
+### **Understand if an item is under the 400KB limit**
+
+```js
 const isValid = CALC.IsUnderLimit(item);
 ```
-```
-true
```

Comment:
This doesn't add any value, IMO. Happy to restore this if anyone disagrees. Same change applies further down.